pages/integrations/azure.md

```diff
@@ -0,0 +1,81 @@
+# [Microsoft Azure IoT Suite][azure] integration
+
+[Resin.io][resin] makes it simple to deploy, update, and maintain code running on remote devices. Microsoft's new IoT Suite makes it easy manage and capture the data those devices generate. This tutorial will walk you through using the two in conjuction. 
+
+## Connect your device to resin.io
+
+Follow our [getting started guide][installing] or watch our screencast to get your device connect to your resin.io dashboard. Once your device shows up on the dashboard your ready for the next step.
+
+![factory-build](/img/integrations/azure/factory-build.png)
+
+## Create the resin.io API Key 
+
+Find your the Applications ID from the dashboard url: https://dashboard.resin.io/apps/NNNN/devices and your auth-token from the [preferences panel](https://dashboard.resin.io/preferences?tab=details). Then combine insert the two into the curl request below, it will return a permanent API key.
+
+```
+curl -H 'Authorization: Bearer AUTH_TOKEN' -X POST https://api.resin.io/application/NNNN/generate-api-key
+```
+
+## Add resin.io API key to IoT Hub Admin portal 
+
+Go to the IoT Hub Admin portal select 'Resin.io Config' and set the `App ID` and the newly generated `API Key`.
```

Comment:
It's important to notice that our integration will not be part of the code microsoft releases, instead it will stay as a separate fork that should be used instead of the official sample solution.